src/modules/newgame.js

```diff
@@ -0,0 +1,16 @@
+export default async function newGame(gameName, url) {
+  const requestURL = url;
+  const request = new Request(requestURL);
+  const data = { name: gameName };
+
+  const response = await fetch(request, {
+    method: 'POST',
+    body: JSON.stringify(data),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+
+  const output = await response.json();
+  return output.result;
+}
```

Comment:
- [ ] Kindly use the arrow function instead of regular functions. [why you should arrow functions.](https://betterprogramming.pub/why-javascript-arrow-functions-are-useful-aa023bad403a)